pkg/apis/autoscaling/v1alpha1/pa_validation.go

```diff
@@ -52,6 +52,7 @@ func (pa *PodAutoscaler) validateMetric() *apis.FieldError {
 		switch pa.Class() {
 		case autoscaling.KPA:
 			switch metric {
+			// TODO(yanweiguo): implement RPS autoscaling for KPA.
```

Comment:
We should be able to implement it in the HPA at the same time as it'll be surfaced by the scraper, right?

Reply:
We need to change the `MetricProvider` or add a new one for HPA. My plan is to enable it for KPA first then HPA. Does it sound reasonable to you? Added a TODO for HPA as well.

Reply:
Yep, sounds reasonable!